dspMetadataGUI/util/utils.py

```diff
@@ -313,13 +313,36 @@ class Datatype(Enum):
     SHORTCODE = 16
     """Shortcode"""
 
+    @classmethod
+    def is_string_like(cls, type) -> bool:
+        """Return True if type is sting or string-like"""
+        if type in [cls.STRING,
+                    cls.STRING_OR_URL,
+                    cls.URL,
```

Comment:
STRING, STRING_OR_URL and URL seems somewhat redundant, can it be reduced to only STRING and URL?

Reply:
generally, yes! The entire implementation of the data model could (and should) be streamlined. However, since this is code that hopefully becomes deprecated soon, I don't plan on doing this at the moment.  
(I did indeed play with the thought of rewriting the entire thing - and even started with the implementation... using a better GUI framework; streamlining the data model; upgrading to model V2; fixing major conceptional flaws in the GUI like not having a language option for strings... but seeing how quickly the web frontend takes shape, the prospect didn't warrant the work necessary.)

And in the current implementation, `STRING_OR_URL` is needed because some props can be either a literal or a URL, which need to be treated differently e.g. in validation. So keeping this ugly little redundancy is simply the easiest way. :)  
But nonetheless you're right. This should be done better!